index.html

```diff
@@ -11,10 +11,11 @@
   gtag('config', 'UA-169665333-3');
 </script>
 </head>
-<body>
+<body style="margin:0px;">
 <div align="center">
     <a href="https://github.com/EESSI"><img src="./img/EESSI_logo_horizontal.png"/></a>
-    <p><em>(click to continue)</em></p>
+    <p style="margin:0px;"><a href="https://github.com/EESSI">github.com/EESSI</a></p>
+    <p><a href="https://eessi.github.io/docs/">documentation</a></p>
```

Comment:
Maybe also link to website (more high-level info) and Twitter account?

See https://eessi.github.io/docs/contact/

Reply:
Good idea. Originally I wanted to have the links in one line nicely spaced but without using &nbsp; . Maybe as a quick hack we can just use &nbsp; and add links to website and twitter account.

Reply:
Maybe just include a `-` in between? And strip the `https://` bits.

```
eessi-hpc.org - github.com/EESSI - eessi.github.io/docs - twitter.com/eessi_hpc
```

(with links behind them)